proposals/NNNN-observed.md

```diff
@@ -0,0 +1,497 @@
+# Transactional Observation of Values
+
+* Proposal: [SE-NNNN](NNNN-observed.md)
+* Authors: [Philippe Hausler](https://github.com/phausler)
+* Review Manager: None
+* Status: Draft
+* Framework: Observation
+
+## Introduction
+
+Observation was introduced to add the ability to observe changes in graphs of 
+objects. The initial tools for observation afforded seamless integration into 
+SwiftUI, however aiding SwiftUI is not the only intent of the module - it is 
+more general than that. This proposal describes a new safe, ergonomic and 
+composable way to observe changes to models using an AsyncSequence, starting 
+transactions at the first willSet and then emitting a value upon that 
+transaction end at the first point of consistency by interoperating with 
+Swift Concurrency.
+
+## Motivation
+
+Observation was designed to allow future support for providing an `AsyncSequence` 
+of values, as described in the initial [Observability proposal](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0395-observability.md). 
+This follow-up proposal offers tools for enabling asynchronous sequences of 
+values, allowing non-SwiftUI systems to have the same level of "just-the-right-amount-of-magic" 
+as when using SwiftUI.
+
+Numerous frameworks in the Darwin SDKs provide APIs for accessing an 
+`AsyncSequence` of values emitted from changes to a property on a given model 
+type. For example, DockKit provides `trackingStates` and Group Activities 
```

Comment:
given that (AFAICT) the cited examples are closed source, it's not self-evident that they have semantics compatible with what's being proposed here. are you asserting that they do – i.e. these API could be re-implemented with the proposed `Observed` type? are there open source examples that could be used here instead?